### PR TITLE
Homepage not logged in

### PR DESCRIPTION
**Pull request by nicole-loescher**
Here is the completed home page for a user that is not logged in. It also has the functionality to allow the user to click on the top films in a genre as well as a random movie generator.

### CODE REVIEW

File: public/stylesheets/index.css [hunk 1]
```diff
@@ -56,16 +61,111 @@ label{
...
+    padding: 15px;
+    margin: 10px auto;
+    border: 1px solid #162f65;
+    border-radius: 5px;
+}
+.bestOfBox2{
+    display:flex;
+    flex-flow: column wrap;
+    height: 135px;
+
```
Comment (genagain) [inline public/stylesheets/index.css]: Please remove this extra line

File: routes/index.js [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
-    token: req.csrfToken(),
-    title: "a/A Express Skeleton Home",
-  });
-});
+router.get('/', csrfProtection, asyncHandler(async(req, res, next) => {
+  const topMovies = await Movie.findAll({where: { voteRating:{ [Op.gt]: 8 }, voteCount: { [Op.gt]: 2000 } }, order: [['voteRating', 'DESC']], limit: 5});
+  const recentReviews = await Review.findAll({ include: [Movie, User], order: [['updatedAt', 'DESC']], limit: 5});
+  function randomNum(max) {
+    return Math.floor(Math.random() * Math.floor(max));
+  };
```
Comment (genagain) [inline routes/index.js]: Nice! Please define this function above the router handler.

File: routes/index.js [hunk 3]
```diff
@@ -1,17 +1,25 @@
...
+  const randomBest = randomNum(880);
```
Comment (genagain) [inline routes/index.js]: Do you know that there will always be 880 movies in the database? If there are ever any fewer then the app would try to query a primary key that doesn't exist

File: views/index.pug [hunk 4]
```diff
@@ -1,11 +1,51 @@
...
+block content
+  h1.title= title
+    p.descriptor See what your friends are watching and rate movies you watch!
+    if !locals.authenticated
+      div.bestOfBox
+        h2 1st Annual Marvelous Movies Random Movie Generator
+        a.submit(href='/movies/' + bestFilm.id) Feeling Lucky?!<br>(click for a random movie)
+        h2 See top Movies by category
+        div.bestOfBox2
+          a(href='/movies/genre/Action') Best Action
```
Comment (genagain) [inline views/index.pug]: Could you pass all of the genres to the `index.pug` template an iterating over them to render these links?

**Review by genagain — CHANGES_REQUESTED** [review]
Looks pretty good! Once you address the changes I requested and/or answer my questions I'll approve it.

**Comment by nicole-loescher** [comment]
Hey Gen, thank you for taking a look. Those requested changes have been made!

**Review by genagain — APPROVED** [review]
Looks good to me! Ship it! 🚀

**Pull request merged**